commitizen/commands/changelog.py

```diff
@@ -45,7 +45,9 @@ def __init__(self, config: BaseConfig, args):
             self.config.settings.get("change_type_map") or self.cz.change_type_map
         )
         self.change_type_order = (
-            self.config.settings.get("change_type_order") or self.cz.change_type_order
+            self.config.settings.get("change_type_order")
+            or self.cz.change_type_order
+            or defaults.change_type_order
```

Comment:
What would happen if you have a custom cz rule, without setting `change_type_order`? I see the `defaults` would be used, but would this explode?
It would always go into this line https://github.com/commitizen-tools/commitizen/blob/master/commitizen/commands/changelog.py#L159

Do we have a test case for that situation?

Reply:
we've not yet have a test case for it. I'll add one. Wondering why do you think it would explode? for the case that users use different change types?